src/pie_core/preparable.py

```diff
@@ -3,8 +3,12 @@
 
 logger = logging.getLogger(__name__)
 
+PREPARED_ATTRIBUTES: List[str] = []
+
 
 class PreparableMixin:
+    """Mixin for preparable classes."""
```

Comment:
Please add a real description for the class (what it is meant for? how can it be used?)

Reply:
@RainbowRivey "what it is meant for?" is still missing